web/sidebar_spec_cli.js

```diff
@@ -3,13 +3,19 @@ module.exports = {
     {
       type: 'category',
       label: 'About',
-      items: ['reference/cli/index', 'reference/cli/supabase-init'],
+      items: ['reference/cli/index'],
       collapsed: true,
     },
     {
       type: 'category',
-      label: 'Local Development',
-      items: ['reference/cli/supabase-start', 'reference/cli/supabase-stop', 'reference/cli/supabase-eject'],
+      label: 'Command reference',
```

Comment:
is this branch out of date from master?

Reply:
Was up to date with master last night when I did it. I can see if I need to merge master back in though?

Reply:
I should have merged master into my branch now